### ISSUE TITLE
Cannot contact Algorand node.

### ISSUE THREAD

**Issue by Bhaney44** [algorand/pyteal#69]
I am working on the [Teal Badge](https://github.com/algorand-devrel/AlgorandDeveloperTasks/blob/master/Tasks/TEALTask.md), and I created a [Repository](https://github.com/Bhaney44/Build-an-Algorand-Smart-Contract) for my work. I am running the following from the command line in my node directory.

`goal clerk compile stateless_swap.teal`

The code is returning the following error.

`Cannot contact Algorand node: open /Users/brianhaney/node/data/algod.net: no such file or directory`

I had tried to run this in the Sandbox, as a solution to the [Issue](https://github.com/algorand/pyteal/issues/68). However, developing in Sandbox led me to many more problems because the command line syntax, as well as the general directory were too convoluted. So, now I am focusing on solving this problem specifically for a node. I'd appreciate any suggestions or advice - thanks!

**Comment by Bhaney44** [comment]
I ran a PyTeal script instead. I ran from the `node` directory.

`python3 tealbadge.py`

The error I am getting is the following.

`/Library/Frameworks/Python.framework/Versions/3.7/Resources/Python.app/Contents/MacOS/Python: can't open file 'tealbadge.py': [Errno 2] No such file or directory`

I then ran the same script from the `desktop` directory and received the following error return.

```
Traceback (most recent call last):
  File "tealbadge.py", line 89, in <module>
    compiled = compileTeal(approval_program(), mode=Mode.Application, version=2)
TypeError: compileTeal() got an unexpected keyword argument 'version'
```
I will continue re-reading the documentation to problem solve.
